### ISSUE TITLE
`html-indent`: extra warning on line with comment

### ISSUE THREAD

**Issue by usernamehw** [vuejs/eslint-plugin-vue#330]
* **ESLint Version:** 4.14.0
* **eslint-plugin-vue Version:** 4.1.0
* **babel-eslint** 8.1.2
* **Node Version:** 8.9.3

```javascript
// this is not full configuration
"parserOptions": {
    "parser": "babel-eslint",
},
"rules": {
    "vue/html-indent": [1, 4, {
        "attribute": 1,
        "closeBracket": 0
    }],
}


```

**What did you do? Please include the actual source code causing the issue.**

```vue
<!-- this code is perfectly valid -->
<template>
    <div>
        <div></div>
    </div>
</template>

<!-- [eslint] Expected indentation of 4 spaces 
but found 8 spaces. (vue/html-indent) -->
<template>
    <div>
        <!-- <div></div> -->
    </div>
</template>




```

**What did you expect to happen?**
For it to treat comment beginning the same way as it does for tag.

**Comment by michalsnik** [comment]
Hey @usernamehw Thanks for posting this issue. There was a recent change in indentation of comments, perhaps this is related to that change. With comments sometimes you want to add information about closing tag, and the other time as you showed you want to treat it like an element. We want to support both cases.

We'll look into this :)

**Comment by usernamehw** [comment]
Actually, there is no need to consider comments at all. I just don't want to see a bunch of warnings.

They could be at the 0 column or at any other. Something like: `/^\s+<!--/` should simply ignore indent.
